server/sonar-server/src/main/java/org/sonar/server/user/index/UserIndexer.java

```diff
@@ -58,52 +61,104 @@ public void indexOnStartup(Set<IndexType> emptyIndexTypes) {
     doIndex(newBulkIndexer(Size.LARGE), null);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "doIndex"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)